sdk/storage/Azure.Storage.Files.Shares/src/Models/ShareAudience.cs

```diff
@@ -0,0 +1,74 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.ComponentModel;
+using Azure.Core;
+
+namespace Azure.Storage.Files.Shares.Models
+{
+    /// <summary>
+    /// Audiences available for Blobs
+    /// </summary>
+    public readonly partial struct ShareAudience : IEquatable<ShareAudience>
+    {
+        private readonly string _value;
+
+        /// <summary>
+        /// Intializes new instance of <see cref="ShareAudience"/>.
+        /// </summary>
+        /// <param name="value">
+        /// The Azure Active Directory audience to use when forming authorization scopes.
+        /// For the Language service, this value corresponds to a URL that identifies the Azure cloud where the resource is located.
+        /// For more information: <see href="https://learn.microsoft.com/en-us/azure/storage/blobs/authorize-access-azure-active-directory" />.
+        /// </param>
+        /// <remarks>Please use one of the static constant members over creating a custom value unless you have specific scenario for doing so.</remarks>
+        public ShareAudience(string value)
+        {
+            Argument.AssertNotNullOrEmpty(value, nameof(value));
+            _value = value;
+        }
+
+        private const string _publicAudience = "https://storage.azure.com/.default";
```

Comment:
This should not contain `/.default` that should be added by the clients, to allow for the clients using more specific permissions in scopes in the future. The audience should just be `https://storage.azure.com`.

Reply:
Done, removed the /.default and the default scope to the audience later.